### PR TITLE
1123 add a short bio of the author before the article akshanjay

### LINKED ISSUE #1123: Add a short bio of the author before the article
Editorial comment:
> "Article pages have a deck with a short bio (written by author)."

See [example article](https://thetyee.ca/Opinion/2023/05/26/Vote-For-UCP-Vote-Wage-Suppression)

### PR DESCRIPTION
**Pull request by FireBoyAJ24**
## What problem does this PR solve?

#1123

## How did you fix the problem?

The design ideas can be seen [here](https://www.figma.com/file/hXJ7SUtqbJHiY7NFMNOYAE/Redesign-of-the-Article-Information-Section?type=design&node-id=0%3A1&mode=design&t=YLvxw9bCqCtURrfF-1).

I would like a review for the desktop version. There are currently issues with the mobile version.

### CODE REVIEW

File: article/models.py [hunk 1]
```diff
@@ -44,6 +44,8 @@
     ObjectList,
     TabbedInterface,
 )
+
+from bs4 import BeautifulSoup
```
Comment (FireBoyAJ24) [inline article/models.py]: Needs to be removed

**Comment by SamuelmdLow** [comment]
There was a lot of overcomplexity in the front and back end. I simplified things and in doing so also fixed an error where only one contributor from each role (author, illustrator, photographer, etc) is credited.

**Review by SamuelmdLow — APPROVED** [review]

**Comment by FireBoyAJ24** [comment]
The magazine template is broken and the author card alignment is a bit all over the place. Plus the published dates and reading time is not present in the wide articles. Therefore, the simplified authors bio need to be reviewed.

image
image

**Review by SamuelmdLow — DISMISSED** [review]
I feel that my reverted changes improved the code. I won't add them back until we can discuss it next meeting

**Review by FireBoyAJ24 — COMMENTED** [review]
The commits after this requested review has tried to their best to solve the issues.

**Review by brittkhat — APPROVED** [review]
lgtm ready to resolve merge conflicts

**Pull request merged**